docs/COMPILATION.MD

```diff
@@ -25,7 +25,13 @@ RHEL:
 ```bash
 yum install -y glew-devel glfw-devel cmake gcc libcurl-devel tesseract-devel leptonica-devel clang
 ```
-Rust 1.54 or above is also required.[Install Rust](https://www.rust-lang.org/tools/install). Check specific compilation methods below, on how to compile without rust
+
+Arch:
+```bash
+sudo paru -S glew glfw curl tesseract leptonica cmake gcc clang
+```
+
+Rust 1.54 or above is also required. [Install Rust](https://www.rust-lang.org/tools/install). Check specific compilation methods below, on how to compile without rust.
```

Comment:
I believe we'll need a higher Rust version any day now :-)

Reply:
All the following code blocks also have the that so I just added it for consistency.